cedar-policy-validator/src/extensions/decimal.rs

```diff
@@ -13,6 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+//! Note on panic safety
+//! This module depends on the cedar parser only constructing ASTs with valid extension calls
+//! If any of the panics in this file are triggered, that means that this file has become
+//! out-of-date with the decimal extension definition in CedarCore.
```

Comment:
I don't think this depends on the AST at all.

The function `extension_schema` is the only place here the `get_argument_types`, `get_return_type` and `get_argument_check` functions are called where they are called on elements of `decimal::extension().funcs()`. So, the panic safety condition is that the we've accounted for every extension function in the match.

If we add a test that just executes `extension_schema()` we can guarantee that these panics don't occur at runtime. This is probably covered by tests elsewhere that construct the extension data structure, but we can retest here.

Reply:
Ok sounds good. I'll make that test